contrib/completions/bash/buildah

```diff
@@ -164,6 +164,8 @@ return 1
          --version -v
          "
      local options_with_args="
+         --registries-conf
```

Comment:
Do we need these here since they're in the global options below?

Reply:
I believe we only need the buildah_buildah() ones, the other global options in buildah() should be removed.

Reply:
updated. It's strange why tests fail.